asammdf/mdf_v4.py

```diff
@@ -433,24 +433,28 @@ def _read(self):
                         channel_group['flags'] &= ~v4c.FLAG_CG_BUS_EVENT
                     else:
                         comment = channel_group.comment.replace(' xmlns="http://www.asam.net/mdf/v4"', '')
-                        comment_xml = ET.fromstring(comment)
-                        can_msg_type = comment_xml.find('.//TX').text.strip(' \t\r\n')
-                        if can_msg_type == 'CAN_DataFrame':
-                            common_properties = comment_xml.find(".//common_properties")
-                            can_id = 1
-                            message_id = -1
-                            for e in common_properties:
-                                name = e.get('name')
-                                if name == 'MessageID':
-                                    if e.get('ci') is not None:
-                                        can_id = int(e.get('ci'))
-                                    message_id = int(e.text)
-                            grp['can_id'] = can_id
-                            grp['message_name'] = message_name
-                            grp['message_id'] = message_id
+                        try:
+                            comment_xml = ET.fromstring(comment)
```

Comment:
At this point the channel group comment should be a valid XML with relevant information about the CAN message, or the bus logging is done for another bus than CAN.

I guess you have already a measurement, so please print:
```python
print(channel_group.acq_name, channel_group.comment)
print(channel_group.acq_source)
```